src/openforms/contrib/kvk/validators.py

```diff
@@ -48,6 +48,7 @@ class KVKBranchNumberValidator(NumericBaseValidator):
 class KVKRemoteBaseValidator:
     query_param = NotImplemented
     value_label = NotImplemented
+    components: tuple = ("textfield",)
```

Comment:
maybe `relevant_for_components` to better cover the meaning? The type hint should also be fully specified.

```py
relevant_for_components: tuple[str] = ("textfield",)
```

Looking at the remaining code, I think we should not specify this as a class attribute, but instead make it a required argument to the `register` decorator so you'd register a validator like this:

```py
@register("kvk-kvkNumber", verbose_name=_("KvK number"), for_components=("textfield",))
class KVKNumberRemoteValidator(...):
     ...
```

This information is not relevant for the validator implementation itself but only for the plugin/registry metadata.